user/articles/170-content-publication-management/03-change-lists/02-enabling-change-lists.markdown

```diff
@@ -0,0 +1,39 @@
+# Enabling Change Lists
+
+To enable the Change Lists feature, follow the steps below:
+
+1.  Navigate to the Control Panel &rarr; *Change Lists* &rarr; *Settings*.
+
+2.  Click the *Yes* selector button.
+
+![Figure 1: You can enable Change Lists from the Control Panel.](../../../images/enabling-change-lists.png)
+
+3.  Save your selection to enable the Change Lists functionality.
+
+    The *Save* button saves your selection. Choosing the *Save and Go to
+    Overview* option saves your selection and also navigates you to the Change
+    Lists overview page.
+
+Your changes are now tracked globally! The current environment will serve as
+your first change list (baseline) until it's published. All
```

Comment:
This is a bit confusing for me. When you turn on the feature, the only change list you have is the production, which is really just a virtual change list as it represents the environment everyone sees currently. You need to create a change set manually to start editing the supported content without showing it to your audience immediately.

Reply:
Good catch; I see what you mean. I'll explain that the production environment is the initial setting and that they should create a new change list to begin leveraging the framework.